src/libraries/System.Text.Json/tests/Serialization/PolymorphicTests.cs

```diff
@@ -9,35 +9,62 @@
 
 namespace System.Text.Json.Serialization.Tests
 {
-    public static class PolymorphicTests
+    public class PolymorphicTests_String : PolymorphicTests
```

Comment:
Is there a way to also add the method overload permutations? Generic call vs one where we pass in type directly?

Reply:
Looks like all the `string` and `Stream` overloads are tested here. 

@steveharter, how about adding the `byte[]` overloads as well (and return `Encoding.UTF8.GetString(result)` in the `Serialize[<T>]` overloads)?